### PR TITLE
Add a more info to the footer

### PR DESCRIPTION
**Pull request by NyakudyaA**
Before
image

After  

image

### CODE REVIEW

File: geodata_mart/templates/components/_footer_default.html [hunk 1]
```diff
@@ -1,8 +1,179 @@
...
+        .footer-menu ul {
+            display: flex;
+            margin-top: 10px;
+            margin-bottom: 20px;
+        }
+    }
+</style>
+<footer class="text-secondary">
+    <div class="footer-content">
+        <h3>Spatial Data Downloader</h3>
```
Comment (seabilwe) [inline geodata_mart/templates/components/_footer_default.html:157]: @NyakudyaA I have put a comment here #101. The name of the site will be Geospatial Data Downloader.

Comment (seabilwe) [inline geodata_mart/templates/components/_footer_default.html:157]: @NyakudyaA please comment here #101. The site will be re-branded to Geospatial Data Downloader.

**Review by seabilwe — CHANGES_REQUESTED** [review]
Change Spatial Data Downloader o Geospatial Data Downloader

File: geodata_mart/templates/components/_footer_default.html (hunk 1 above)
Comment (NyakudyaA) [reply geodata_mart/templates/components/_footer_default.html:157]: Thanks, will fix in an upcoming commit

**Pull request merged**